vcr/verifier/interface.go

```diff
@@ -48,8 +48,8 @@ type Verifier interface {
 	RegisterRevocation(revocation credential.Revocation) error
 
 	// VerifyVP verifies the given Verifiable Presentation. If successful, it returns the credentials within the presentation.
-	// If verifyVCs is true, it will also verify the credentials inside the VP, checking their correctness, signature and trust status.
-	VerifyVP(presentation vc.VerifiablePresentation, verifyVCs bool, validAt *time.Time) ([]vc.VerifiableCredential, error)
+	// If verifyVCs is true, it will also verify the credentials inside the VP, checking their correctness, signature and trust status (if allowUntrustedVCs is true).
```

Comment:
```suggestion
	// If verifyVCs is true, it will also verify the credentials inside the VP, checking their correctness, signature and trust status (unless allowUntrustedVCs is true).
```